CONTRIBUTING.rst

```diff
@@ -1,6 +0,0 @@
-Contributing
-============
-
-Thanks for your interest! Please read `our contributing guideliness
-<http://django-freeradius.readthedocs.io/en/latest/general/contributing.html>`_
-and submit a PR.
```

Comment:
hey guys, we should not delete this file, as this is shown as a link by github when opening new issues (now they moved to the bottom right corner of the open issue page)

Reply:
Okay, I will restore this.